mw-webapp/src/component/editableText/EditableText.tsx

```diff
@@ -116,10 +122,22 @@ export const EditableText = <T extends string | number>(props: EditableTextProps
   };
 
   /**
-   * Update value
+   * Check is value valid or not according min length
    */
-  const updateValue = (updatedValue: string | number) => {
-    setValue(updatedValue as T);
+  const getValidValue = (updateValue: string | number) => {
+    const isInvalidLabelLength = typeof updateValue === "string" &&
```

Comment:
This variable exist inside Editable component so name is not clear. Let's rename to isInvalidTextLength

Reply:
done